application/wrapper/storage.go

```diff
@@ -20,8 +20,14 @@ func NewStorage(storageInter dynamics.StorageGetter) *Storage {
 }
 
 // GetMaxBytes returns MaxBytes
-func (s *Storage) GetMaxBytes() uint32 {
-	return s.storage.GetMaxBytes()
+func (s *Storage) GetMaxBytes() (uint32, error) {
+	if s == nil {
```

Comment:
nit: It's generally considered an acceptable practice to treat nil pointer receivers as an edge case when a `New` method is supplied for them. In this case it'll let you just handle the return of `uint32` and have one less error path to check in calling code (which would cause a panic and get caught anyways).